Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Build GoChain in a stock Go builder container
 FROM golang:1.13-alpine as backend_builder
-RUN apk --no-cache add build-base git bzr mercurial gcc linux-headers g++ make
+RUN apk --no-cache add build-base git mercurial gcc linux-headers g++ make
```

Comment:
To fix https://circleci.com/gh/gochain/explorer/1625
![Screenshot from 2020-07-02 07-28-47](https://user-images.githubusercontent.com/1194128/86358788-b38ef300-bc35-11ea-952e-45119dc2eb4f.png)
I don't understand why it's no longer available, but it was also not necessary so :shrug: